sem-dockerize

```diff
@@ -9,10 +9,8 @@ run::dockerize(){
 
   if [[ $? -ne 0 ]]; then
     echo "Failed to start service $service on port $port"
-    echo "toolbox_sem_service_error_rate_${service} 1" >> /tmp/toolbox_metrics
-    exit 1
+    echo 1
```

Comment:
Should this be exit?

Reply:
You're right. Fixed the logic and removed the output, it is handled later in sem-version anyway.